persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/GenePanelMapper.xml

```diff
@@ -0,0 +1,302 @@
+        <collection property = "genes" ofType="org.cbioportal.model.Gene">
+            <result property="entrezGeneId" column="entrezGeneId"/>
+            <result property="hugoGeneSymbol" column="hugoGeneSymbol"/>
+            <result property="type" column="type"/>
+            <result property="cytoband" column="cytoband"/>
+            <result property="length" column="length"/>
+        </collection>
+    </resultMap>
+    
+   <resultMap id="genePanelNoGenesResultMap" type="org.cbioportal.model.GenePanel">
+        <result property="stableId" column="stableId"/>
+        <result property="description" column="description"/>
+    </resultMap>
+    
+
+   <resultMap id="sampleResultMap" type="org.cbioportal.model.Sample">
+        <result property="internalId" column="INTERNAL_ID"/>
+        <result property="stableId" column="STABLE_ID"/>
+        <result property="patientId" column="PATIENT_ID"/>
+        <result property="typeOfCancerId" column="TYPE_OF_CANCER_ID"/>
+        <association property = "patient" javaType="org.cbioportal.model.Patient">
+            <result property="internalId" column="INTERNAL_ID"/>
+            <result property="stableId" column="STABLE_ID"/>
+            <result property="cancerStudyId" column="CANCER_STUDY_ID"/>
+            <association property="cancerStudy" javaType="org.cbioportal.model.CancerStudy">
+                <result property="cancerStudyId" column="CANCER_STUDY_ID"/>
+                <result property="cancerStudyIdentifier" column="CANCER_STUDY_IDENTIFIER"/>
+                <result property="typeOfCancerId" column="TYPE_OF_CANCER_ID"/>
+                <result property="name" column="NAME"/>
+                <result property="shortName" column="SHORT_NAME"/>
+                <result property="description" column="DESCRIPTION"/>
+                <result property="publicStudy" column="PUBLIC"/>
+                <result property="pmid" column="PMID"/>
+                <result property="citation" column="CITATION"/>
+                <result property="groups" column="GROUPS"/>
+                <result property="status" column="STATUS"/>
+                <association property="typeOfCancer" javaType="org.cbioportal.model.TypeOfCancer">
+                    <result property="typeOfCancerId" column="TYPE_OF_CANCER_ID"/>
+                    <result property="name" column="NAME"/>
+                    <result property="clinicalTrialKeywords" column="CLINICAL_TRIAL_KEYWORDS"/>
+                    <result property="dedicatedColor" column="DEDICATED_COLOR"/>
+                    <result property="shortName" column="SHORT_NAME"/>
+                    <result property="parent" column="PARENT"/>
+                </association>
+            </association>
+        </association>
+        <association property="typeOfCancer" javaType="org.cbioportal.model.TypeOfCancer">
+            <result property="typeOfCancerId" column="TYPE_OF_CANCER_ID"/>
```

Comment:
@zheins: could you explain how the framework knows that this one refers to the `sample.TYPE_OF_CANCER_ID` in the result set, and not to one of the other two, `cancer_study.TYPE_OF_CANCER_ID` and `type_of_cancer.TYPE_OF_CANCER_ID`? And likewise for `sample.STABLE_ID` and `patient.STABLE_ID`.